### PR TITLE
[full-ci] Use new icons and bump ODS to 12.0.0(-alpha) in accounts/settings

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Necessary updates in accounts & settings UIs since we're changing available icons in the ODS

## Checklist:
- Perhaps needs a `web` commit ID bump after https://github.com/owncloud/web/pull/6142 is merged

## TODO:
- [ ] settings ui should get this icon https://github.com/Remix-Design/RemixIcon/blob/master/icons/Design/tools-line.svg according to https://github.com/owncloud/web/issues/6256

### CODE REVIEW

**Comment by pascalwengerter** [comment]
Please don't merge before the referenced web PR got merged and the new `web` commit ID got referenced here, otherwise the icons in accounts/settings don't work with the currently bundled `web` version 🚫

**Review by kulmann — CHANGES_REQUESTED** [review]
IMO this should only get merged when we have it updated to the upcoming web 5.0.0. Good to have an early CI check though. Thanks for that already!

**Comment by kulmann** [comment]
Superseded by https://github.com/owncloud/ocis/pull/3157

**Pull request closed without merging**
